sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -8,13 +8,19 @@
 - Added `SyncPoller<PhoneNumberOperation, Void> beginPurchasePhoneNumbers(String searchId)` in PhoneNumbersClient.
 - Added `SyncPoller<PhoneNumberOperation, Void> beginReleasePhoneNumber(String phoneNumber)` in PhoneNumbersClient.
```

Comment:
Are these added in beta.7? If so, I think your changes should not go to the breaking changes, right?

Reply:
These were new variations of the methods that were added in beta.7 as per suggestions in the API View. :)
They introduce a variation without context (compared to `PollerFlux<PhoneNumberOperation, PurchasePhoneNumbersResult> beginPurchasePhoneNumbers(String searchId, Context context)`)